src/sidebar/components/annotation-header.js

```diff
@@ -34,13 +37,26 @@ function AnnotationHeader({
             {replyCount} {replyPluralized}
           </a>
         </div>
-        {!isEditing && annotation.updated && (
+        {!isEditing && annotation.created && (
```

Comment:
Oh, at some point it might make sense to break off this whole chunk as a sub-component. Didn't quite tip the scales for me just yet.